torch/csrc/utils/python_arg_parser.h

```diff
@@ -435,6 +437,13 @@ inline at::MemoryFormat PythonArgs::toMemoryFormat(int i) {
   return memory_format->memory_format;
 }
 
+inline at::QScheme PythonArgs::toQScheme(int i) {
+  if (!args[i]) return at::kPerTensorAffine;
+  AT_CHECK(THPQScheme_Check(args[i]), "qscheme arg must be an instance of the torch.qscheme");
```

Comment:
@jerryzh168, this is giving deprecated warnings during compilation. Can you change it to TORCH_CHECK?

Reply:
oh sorry, will do.